volumes/miovision/sql/views/create-view-monitor_intersection_movements.sql

```diff
@@ -0,0 +1,59 @@
+CREATE OR REPLACE VIEW miovision_api.monitor_intersection_movements AS (
```

Comment:
when I select from the view, there was one row where sum_volume > intersection_classification_total which might be sus.
![image](https://github.com/user-attachments/assets/0c0dc9e1-b9b2-4fe1-9056-4a0cd9ef1641)

Reply:
Not what I exepcted! It's because `SELECT now()::date AT TIME ZONE 'Canada/Eastern' - interval ' 100 days'` results in `2024-05-21 20:00:00` so the larger value had an extra 4 hrs of data.